tests/run.sh

```
@@ -299,5 +299,90 @@ else
+python3 - "$FOUNDATION_CSS" >"$PARSE_LOG" 2>&1 <<'PYEOF'
+import sys, tinycss2
+css = open(sys.argv[1]).read()
+rules, _ = tinycss2.parse_stylesheet_bytes(css.encode())
+errors = [r for r in rules if r.type == "error"]
+if errors:
+    for e in errors:
+        print(f"  parse error: {e}")
+    sys.exit(1)
+print("OK")
+PYEOF
+if [ $? -ne 0 ]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?.

(SC2181)